next.clayui.com/content/docs/css/components/sidebar.md

```diff
@@ -0,0 +1,464 @@
+---
+title: "Sidebar"
+description: "A vertical panel that appears from the right or the left side of the screen, triggered by a button."
+---
+
+<div class="nav-toc">
+
+- [Structure](#structure)
+- [Variations](#variation)
+    - [Sidebar Light](#sidebar-light)
+
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/sidebar/">Sidebar Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
+</div>
+
+# Structure
+
+Sidebar is an opinionated container to display related content.
+
+Is composed by:
+
+<div class="sidebar-header">
+    Sidebar Header
+</div>
+
+<div class="sidebar-body">
+    Sidebar Body
+</div>
+
+<div class="sidebar-footer">
+    Sidebar Footer
+</div>
+
+# Variations
+
+## Sidebar Light
+
+> Just add `sidebar-light` class on the same element that you are using `sidebar`.
```

Comment:
I dont think we need to make this a quote with `>`

Reply:
Fixed!